doc/input-output-reference/src/overview/group-condenser-equipment.tex

```diff
@@ -139,7 +139,7 @@ \subsubsection{Inputs}\label{inputs-007}
 
 \paragraph{Field: Basin Heater Setpoint Temperature}\label{field-basin-heater-setpoint-temperature}
 
-This numeric field contains the set point temperature (˚C) for the basin heater described in the previous field. The basin heater is active when the outdoor air dry-bulb temperature falls below this setpoint temperature, as long as the tower fan is off and water is not flowing through the tower. This set point temperature must be greater than or equal to 2˚C, and the default value is 2˚C if this field is left blank.
+This numeric field contains the set point temperature (°C) for the basin heater described in the previous field. The basin heater is active when the outdoor air dry-bulb temperature falls below this setpoint temperature, as long as the tower fan is off and water is not flowing through the tower. This set point temperature must be greater than or equal to 2˚C, and the default value is 2˚C if this field is left blank.
```

Comment:
It's kinda hard to see here, but did this also need the other degree symbols in this line to be replaced?

Reply:
And if so, there is a conflict in the doc cmake rules.  I'm happy to help resolve those and address this at the same time, if it needs to be done.  Otherwise I can just resolve the conflicts.

Reply:
I'll fix it up now.